app/src/molecules/MediaContainerContent/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { useTranslation } from 'react-i18next'
+import { useDispatch } from 'react-redux'
+
+import { Chip, StyledText } from '@opentrons/components'
+
+import { Skeleton } from '/app/atoms/Skeleton'
+import { cameraPhotoOpenAction } from '/app/redux/shell'
+
+import styles from './media.module.css'
+
+import type { ReactNode } from 'react'
+
+export interface MediaContainerContentProps {
+  mediaContent: ReactNode
+  centerPrimaryText: string
+  centerSecondaryText: string
+  rightPrimaryText: string
+  imagePath: string
+  robotName: string | null
+  state: 'loading' | 'error' | null
+  overflowMenu: JSX.Element | null
+  hoverText: string | null
+  isCurrentCommandError: boolean | null
+}
+
+export function MediaContainerContent(
+  props: MediaContainerContentProps
+): JSX.Element {
+  const {
+    mediaContent,
+    centerPrimaryText,
+    centerSecondaryText,
+    rightPrimaryText,
+    state,
+    robotName,
+    overflowMenu,
+    imagePath,
+    isCurrentCommandError,
+  } = props
+  const { t } = useTranslation(['run_details', 'branded'])
+  const dispatch = useDispatch()
+  const isLoading = state === 'loading' || state === 'error'
+  const onClick = (): void => {
+    if (isLoading) return
```

Comment:
```suggestion
    if (isLoading) {
    return
    }
```
I know it seems nitty, but always including braces after conditional statements prevents the occasional bug, especially when someone else comes in and refactors the conditional statement later on.